commands/show-dns-fqdn-ips.go

```diff
@@ -0,0 +1,136 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/activecm/rita/pkg/data"
+	"github.com/activecm/rita/pkg/hostname"
+	"github.com/activecm/rita/resources"
+	"github.com/olekukonko/tablewriter"
+	"github.com/urfave/cli"
+)
+
+func init() {
+	command := cli.Command{
+		Name:      "show-dns-fqdn-ips",
+		Usage:     "Print IPs associated with FQDN via DNS",
+		ArgsUsage: "<database> <fqdn>",
+		Flags: []cli.Flag{
+			ConfigFlag,
+			humanFlag,
+			delimFlag,
+			netNamesFlag,
+		},
+		Action: showFqdnIps,
+	}
+
+	bootstrapCommands(command)
+}
+
+func showFqdnIps(c *cli.Context) error {
+	db := c.Args().Get(0)
+	if db == "" {
+		return cli.NewExitError("Specify a database", -1)
+	}
+
+	f := c.Args().Get(1)
```

Comment:
Please rename this `fqdn` and remove line 42 since these variables are assigned the same value.